Automattic/woocommerce-payments
# Update Discover and Diners logos

Fixes #8580

#### Changes proposed in this Pull Request
This PR updates the logos to the latest version for Discover and Diners. Additionally, it replaces the Diners logo on the connect page with the more popular iDeal.
<img width="643" alt="Screenshot 2024-04-11 at 14 43 25" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/4bf1dbec-82c4-4214-8a0e-42c23067cfb4">

#### Testing instructions
- Navigate to Onboarding page.
- Verify the Diners logo is replace with the iDeal logo

-------------------


**Post merge**
